stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/viewmodel/messages/MessageListItemLiveData.kt

```diff
@@ -408,3 +425,7 @@ public fun Message.getCreatedAtOrThrow(): Date {
     val created = createdAt ?: createdLocallyAt
     return checkNotNull(created) { "a message needs to have a non null value for either createdAt or createdLocallyAt" }
 }
+
+public fun Message.getCreatedAtOrNull(): Date? {
```

Comment:
Should this live here? Maybe in ui-common? or somewhere else?

Reply:
It is in ui-common but it should be under extensions, will move it there.